### ISSUE TITLE
Feature Request: "legends" for labels

### ISSUE THREAD

**Issue by gdevenyi** [BIC-MNI/Display#19]
As a secondary CSV or similar, allow "names" for label numbers to improve visualization/labelling

**Comment by rdvincent** [comment]
The MINC 2 format actually has the capability to assign symbolic names to voxel values. Perhaps we need to think about making this available in the Display UI.

**Comment by gdevenyi** [comment]
Is this exposed in any command-line programs?

**Comment by rdvincent** [comment]
Not really. It's one of those features we added but didn't really exploit. Might need to revive it.

**Comment by gdevenyi** [comment]
It would be very helpful for us, as we publish label files and having the names embedded in the minc would be really helpful.
